include/linux/soundwire/sdw.h

```diff
@@ -632,6 +632,19 @@ struct sdw_slave {
 
 #define dev_to_sdw_dev(_dev) container_of(_dev, struct sdw_slave, dev)
 
+/**
+ * struct sdw_master_device - SoundWire 'Master Device' representation
+ * @dev: Linux device for this Master
+ * @bus: Bus handle shortcut to improve readability (same as container_of)
```

Comment:
it's not the same as container_of, since the sdw_master_device is a pointer, not a structure in struct sdw_bus().
You cannot use container_of to go from md to bus without this link.

Reply:
Ah, yes. Thanks.